app/forms/idv/inherited_proofing/base_form.rb

```diff
@@ -64,6 +62,14 @@ def user_pii
 
       attr_writer :payload_hash
 
+      def form_response
+        FormResponse.new(
+          success: valid?,
+          errors: errors,
+          extra: {},
+        )
+      end
+
```

Comment:
Don't need to do this if we're _NOT_ sending throttle-stopping errors via extra -> proofing_results -> exception